packages/@okta/vuepress-site/code/vue/okta_vue/index.md

```diff
@@ -14,6 +14,8 @@ This guide will walk you through integrating authentication into a Vue app with
 6. [Connect the Routes](#connect-the-routes)
 7. [Start Your App](#start-your-app)
 
+> This guide is for `@okta/okta-auth-js` v4.5.0.
```

Comment:
This should mention the version of okta-vue as well

Reply:
This guide does not use `okta-vue`

Reply:
how about provide a version range here? like `>= 4.5.0 and <5.0.0`